README.md

```diff
@@ -107,8 +139,9 @@ package. In the code you'll see a call to:
 
 which will get the connection info from the bindings if it is available and
 return it in the format required by the Kafka client.
-                                                                                        122,1         Bot
+122,1 Bot
```

Comment:
What is this line?

Reply:
I think its not necessary. I'm removing it.